extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -0,0 +1,97 @@
+const { usersBucket, mpuBucketPrefix } = require('arsenal').constants;
+const QueuePopulatorExtension =
+    require('../../lib/queuePopulator/QueuePopulatorExtension');
+const LIFECYCLE_BUCKETS_ZK_PATH = '/data/buckets';
```

Comment:
Maybe a refactor in the future, but maybe we can move this to constants file since the conductor also uses this end path